
CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.
To initialize your shell, run

    $ conda init <SHELL_NAME>

Currently supported shells are:
  - bash
  - fish
  - tcsh
  - xonsh
  - zsh
  - powershell

See 'conda init --help' for more information and options.

IMPORTANT: You may need to close and restart your shell after running 'conda init'.


cat: write error: Broken pipe
2023-05-01 16:21:32.911287
2023-05-01 16:21:33.814396
2023-05-01 16:21:34.713511

real	0m2.779s
user	0m2.548s
sys	0m0.199s
cat: write error: Broken pipe
2023-05-01 16:21:45.925545
2023-05-01 16:21:46.720620
2023-05-01 16:21:47.502759

real	0m2.494s
user	0m2.318s
sys	0m0.153s
cat: write error: Broken pipe
2023-05-01 16:21:58.781976
2023-05-01 16:21:59.564096
2023-05-01 16:22:00.308731

real	0m2.506s
user	0m2.304s
sys	0m0.172s
cat: write error: Broken pipe
2023-05-01 16:22:11.575509
2023-05-01 16:22:12.395575
2023-05-01 16:22:13.129696

real	0m2.520s
user	0m2.326s
sys	0m0.171s
cat: write error: Broken pipe
2023-05-01 16:22:24.454377
2023-05-01 16:22:25.411671
2023-05-01 16:22:26.262112

real	0m2.839s
user	0m2.639s
sys	0m0.176s
cat: write error: Broken pipe
2023-05-01 16:22:37.615614
2023-05-01 16:22:40.690302
2023-05-01 16:22:43.378651

real	0m6.826s
user	0m6.442s
sys	0m0.349s
cat: write error: Broken pipe
2023-05-01 16:22:54.614445
2023-05-01 16:22:56.008702
2023-05-01 16:22:58.491685

real	0m4.832s
user	0m4.457s
sys	0m0.338s
cat: write error: Broken pipe
2023-05-01 16:23:09.673615
2023-05-01 16:23:10.035349
2023-05-01 16:23:10.311625

real	0m1.527s
user	0m1.369s
sys	0m0.136s
mkdir: cannot create directory ‘SCODE_D_2’: File exists
┌ Warning: `CSV.read(input; kw...)` is deprecated in favor of `using DataFrames; CSV.read(input, DataFrame; kw...)
│   caller = ip:0x0
└ @ Core :-1
┌ Warning: Assignment to `WZ` in soft scope is ambiguous because a global variable by the same name exists: `WZ` will be treated as a new local. Disambiguate by using `local WZ` to suppress this warning or `global WZ` to assign to the existing global variable.
└ @ ~/stephenz/locaTE-paper/tools/SCODE/SCODE.jl:69
┌ Warning: `writeheader=false` is deprecated in favor of `header=false`
│   caller = ip:0x0
└ @ Core :-1
a
mkdir: cannot create directory ‘SCODE_D_4’: File exists
┌ Warning: `CSV.read(input; kw...)` is deprecated in favor of `using DataFrames; CSV.read(input, DataFrame; kw...)
│   caller = ip:0x0
└ @ Core :-1
┌ Warning: Assignment to `WZ` in soft scope is ambiguous because a global variable by the same name exists: `WZ` will be treated as a new local. Disambiguate by using `local WZ` to suppress this warning or `global WZ` to assign to the existing global variable.
└ @ ~/stephenz/locaTE-paper/tools/SCODE/SCODE.jl:69
┌ Warning: `writeheader=false` is deprecated in favor of `header=false`
│   caller = ip:0x0
└ @ Core :-1
a
mkdir: cannot create directory ‘SCODE_D_6’: File exists
┌ Warning: `CSV.read(input; kw...)` is deprecated in favor of `using DataFrames; CSV.read(input, DataFrame; kw...)
│   caller = ip:0x0
└ @ Core :-1
┌ Warning: Assignment to `WZ` in soft scope is ambiguous because a global variable by the same name exists: `WZ` will be treated as a new local. Disambiguate by using `local WZ` to suppress this warning or `global WZ` to assign to the existing global variable.
└ @ ~/stephenz/locaTE-paper/tools/SCODE/SCODE.jl:69
┌ Warning: `writeheader=false` is deprecated in favor of `header=false`
│   caller = ip:0x0
└ @ Core :-1
a
mkdir: cannot create directory ‘SCODE_D_8’: File exists
┌ Warning: `CSV.read(input; kw...)` is deprecated in favor of `using DataFrames; CSV.read(input, DataFrame; kw...)
│   caller = ip:0x0
└ @ Core :-1
┌ Warning: Assignment to `WZ` in soft scope is ambiguous because a global variable by the same name exists: `WZ` will be treated as a new local. Disambiguate by using `local WZ` to suppress this warning or `global WZ` to assign to the existing global variable.
└ @ ~/stephenz/locaTE-paper/tools/SCODE/SCODE.jl:69
┌ Warning: `writeheader=false` is deprecated in favor of `header=false`
│   caller = ip:0x0
└ @ Core :-1
a
/data/gpfs/projects/punim0638/stephenz/miniconda3/lib/python3.10/site-packages/Scribe-0.0.0-py3.10.egg/Scribe/causal_network.py:18: UserWarning:  WARNING: No expression data argument given. if you intend to load the data from a file, call the method 'read_expression_file'.
  if expression==None: warnings.warn(" WARNING: No expression data argument given. if you intend to load the data from a file, call the method 'read_expression_file'.")
/data/gpfs/projects/punim0638/stephenz/miniconda3/lib/python3.10/site-packages/Scribe-0.0.0-py3.10.egg/Scribe/causal_network.py:298: FutureWarning: Could not cast to int64, falling back to object. This behavior is deprecated. In a future version, when a dtype is passed to 'DataFrame', either all columns will be cast to that dtype, or a TypeError will be raised
  max_rdi_delay = pd.DataFrame({node_id: [np.nan for i in self.node_ids] for node_id in self.node_ids}, index=self.node_ids,
Create causal_model successfully
  Activating project at `/data/gpfs/projects/punim0638/stephenz/locaTE-paper/data/simulated/Synthetic_1000/dyn-BFC/dyn-BFC-1000-7/pidc_env`
    Updating registry at `~/.julia/registries/General.toml`
   Resolving package versions...
  No Changes to `/data/gpfs/projects/punim0638/stephenz/locaTE-paper/data/simulated/Synthetic_1000/dyn-BFC/dyn-BFC-1000-7/pidc_env/Project.toml`
  No Changes to `/data/gpfs/projects/punim0638/stephenz/locaTE-paper/data/simulated/Synthetic_1000/dyn-BFC/dyn-BFC-1000-7/pidc_env/Manifest.toml`
Getting nodes...
Inferring network...
